### PR TITLE
feat: Adding rename selection feature

### PR DESCRIPTION
**Pull request by erickzanardo**
https://github.com/user-attachments/assets/9c295ca5-8520-4e9b-bb10-21fe43262f9e

### CODE REVIEW

File: fire_atlas_editor/pubspec.yaml [hunk 1]
```diff
@@ -16,7 +16,10 @@ dependencies:
   file_selector_macos: ^0.9.0+4
   file_selector_windows: ^0.9.1+4
   flame: ^1.18.0
-  flame_fire_atlas: ^1.5.3
+  flame_fire_atlas:
+    git:
+      url: https://github.com/flame-engine/flame.git
+      path: packages/flame_fire_atlas
```
Comment (spydon) [inline fire_atlas_editor/pubspec.yaml:22]: Why this change?

Comment (spydon) [reply fire_atlas_editor/pubspec.yaml:22]: Oh lol right, I forgot fire_atlas doesn't live in the monorepo

**Review by spydon — APPROVED** [review]

File: fire_atlas_editor/pubspec.yaml (hunk 1 above)
Comment (erickzanardo) [reply fire_atlas_editor/pubspec.yaml:22]: Yeah haha

There is no issue in merging like this for now, but I will publish a new version of the editor only when we publish a new flame fire atlas version on pub
